code/game/objects/effects/effect_system/particle_effects.dm

```diff
@@ -22,17 +22,13 @@
 	var/life = 15
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 
-/obj/effect/particle_effect/water/initialize_pass_flags(datum/pass_flags_container/PF)
+/obj/effect/particle_effect/water/initialize_pass_flags(datum/pass_flags_container/pass_flags)
 	..()
-	if (PF)
-		PF.flags_pass = PASS_THROUGH|PASS_OVER|PASS_MOB_THRU|PASS_UNDER
+	if (pass_flags)
+		pass_flags.flags_pass = PASS_THROUGH|PASS_OVER|PASS_MOB_THRU|PASS_UNDER
 
 /obj/effect/particle_effect/water/Move(turf/newloc)
-	//var/turf/T = src.loc
-	//if (istype(T, /turf))
-	// T.firelevel = 0 //TODO: FIX
-	if (--src.life < 1)
-		//SN src = null
+	if (--life < 1)
```

Comment:
would you mind redoing this so that it doesn't need to use `--` before the var it's modifying